modules/common/src/main/java/StringUtils.java

```diff
@@ -149,4 +149,9 @@ public static final int levenshtein(final String s1, final String s2) {
 
         return v0[len2];
     }
+
+    public static String ignoreBetweenSquareBrackets(final String s) {
+        return s.replaceAll("\\[(.*)\\]", "");
```

Comment:
1.  Use a [static compiled regex](https://www.scala-lang.org/api/2.12.5/scala/util/matching/Regex.html).  2. Gate it behind a simple find of "[" for performance reasons.  There is one lila server and it has a lot to do so if we can save it cycles compiling and performing regexes, then we should do that at every opportunity.  Other than that, looks like you've got yourself a fix!  Well done.

Reply:
By #2, I mean 99.99% of the time we can just return s unmodified after not finding '[' (no regex involved).

Reply:
Got it! Thank you for your feedback.